.github/workflows/darwin-i386.yml

```diff
@@ -0,0 +1,57 @@
+name: darwin-i386
```

Comment:
Let's rename darwin-i386 to darwin-x86-64. I suppose, macos-latest runs on x86-64, according to  https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md. Also we should rename .yaml file and job name too.